src/components/TextInputWithPrefix/index.android.js

```diff
@@ -51,7 +49,14 @@ const TextInputWithPrefix = props => (_.isEmpty(props.prefixCharacter)
                 {..._.omit(props, ['prefixCharacter', 'errorText', 'onChangeText'])}
             />
         </View>
-    ));
+        {!_.isEmpty(props.errorText) && (
+            <InlineErrorText>
+                {props.errorText}
+            </InlineErrorText>
+        )}
+    </>
+
```

Comment:
Remove extra line

Reply:
Dangit, our linter should really catch this. Fixed!